### DISCUSSION TITLE
Allocate node to single user

### DISCUSSION THREAD

**Discussion by Prophet731** [pterodactyl/panel#3575] [Feature Requests]
So I don't know if this idea has been explored yet but would it be possible in an update to allow the ability to assign a node to a user where they can "deploy" servers on their own to that node only and not any other ones. Also, prevent other servers that are not owned by said user from provisioning against it.

The idea behind that is for let's say Minecraft and I would like to offer though WHMCS a "dedicated" option and they would be able to create servers themselves with the block of resources but still using the Pterodactyl portal.

**Comment by matthewpi** [comment]
https://github.com/pterodactyl/panel/issues/3532

This is functionality we are not interested in supporting.
